tests/run/cpp_stl_algo_modifying_sequence_ops.pyx

```diff
@@ -316,17 +316,19 @@ def test_swap_ranges():
     print(a, b)
 
 
-def selection_sort(vector[int] values):
+def selection_sort(vector[int] values, reversed=False):
     """
-    Test iter_swap using cppreference example.
+    Test iter_swap using cppreference example. Extra "reversed argument tests max_element
 
     >>> selection_sort([-7, 6, 2, 4, -1, 6, -9, -1, 2, -5, 10, -9, -5, -3, -5, -3, 6, 6, 1, 8])
     [-9, -9, -7, -5, -5, -5, -3, -3, -1, -1, 1, 2, 2, 4, 6, 6, 6, 6, 8, 10]
+    >>> selection_sort([-7, 6, 2, 4, -1, 6, -9, -1, 2, -5, 10, -9, -5, -3, -5, -3, 6, 6, 1, 8], reversed)
```

Comment:
Took me a while to understand why this worked. Maybe you can figure it out, too. :)
```suggestion
    >>> selection_sort([-7, 6, 2, 4, -1, 6, -9, -1, 2, -5, 10, -9, -5, -3, -5, -3, 6, 6, 1, 8], reversed=True)
```